RxCocoa/Traits/SharedSequence/SharedSequence+Operators.swift

```diff
@@ -108,16 +108,17 @@ extension SharedSequenceConvertibleType {
      Invokes an action for each event in the observable sequence, and propagates all observer messages through the result sequence.
 
      - parameter onNext: Action to invoke for each element in the observable sequence.
+     - parameter onAfterNext: Action to invoke for each element after the observable has passed an onNext event along to its downstream.
      - parameter onCompleted: Action to invoke upon graceful termination of the observable sequence.
      - parameter onSubscribe: Action to invoke before subscribing to source observable sequence.
      - parameter onSubscribed: Action to invoke after subscribing to source observable sequence.
      - parameter onDispose: Action to invoke after subscription to source observable has been disposed for any reason. It can be either because sequence terminates for some reason or observer subscription being disposed.
      - returns: The source sequence with the side-effecting behavior applied.
      */
-    public func `do`(onNext: ((E) -> Void)? = nil, onCompleted: (() -> Void)? = nil, onSubscribe: (() -> Void)? = nil, onSubscribed: (() -> Void)? = nil, onDispose: (() -> Void)? = nil)
+    public func `do`(onNext: ((E) -> Void)? = nil, onAfterNext: ((E) -> Void)? = nil, onCompleted: (() -> Void)? = nil, onSubscribe: (() -> Void)? = nil, onSubscribed: (() -> Void)? = nil, onDispose: (() -> Void)? = nil)
```

Comment:
Let's maybe call this just `afterNext` and add `afterCompleted` and `afterError` and add tests.

If you can make this change in less than one week, we can ship this in 5.0

Reply:
Done.